pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -117,8 +117,9 @@ var (
 			ConfigNamespace: "not-default",
 			IstioVersion:    "1.4",
 		},
-		ConfigNamespace: "not-default",
-		WorkloadLabels:  labels.Collection{{"istio": "ingressgateway"}},
+		ConfigNamespace:     "not-default",
+		WorkloadLabels:      labels.Collection{{"istio": "ingressgateway"}},
+		GatewayClusterNames: map[string]bool{},
```

Comment:
use `map[string]struct{}` not bool. Its slightly more efficient but also more clear